plaza/main_windows.go

```diff
@@ -74,7 +74,7 @@ func sendShellInfo() {
 		Pid      int    `json:"pid"`
 	}{}
 
-	mes.Username = splt[len(splt)-1]
+	mes.Username = "Administrator" //splt[len(splt)-1]
```

Comment:
I'm not sure we want to commit that.

Maybe it's more elegant to send "Administrator" from backend rather than hard coded it there